jsonld2compactjsonldldj/jsonld2compactjsonldldj.py

```diff
@@ -3,10 +3,38 @@
 import argparse
 import json
 import sys
-
+import requests
+from multiprocessing import Pool,current_process
 from pyld import jsonld
 
-
+def init_mp(m,c,rf,url):
+    global mp
+    global context
+    global context_url
+    global record_field
+    mp=m
+    context=c
+    if url:
+        context_url=url
+    else:
+        context_url=None
+    if rf:
+        record_field=rf
+    else:
+        record_field=None
+
+def compact_object(jsonobject):
+    if isinstance(jsonobject, dict):
+        if (record_field and record_field in jsonobject) or (record_field is None):
+            compacted = jsonld.compact(jsonobject, context,  {'skipExpansion': True})
+            if context_url:
+                compacted['@context'] = context_url
+            if mp:
+                with open(str(current_process().name)+"-records.ldj","a") as fileout:           ###avoid raceconditions
```

Comment:
I would recommend to make use of another file naming here (or leave this file name as default (which should then be described somewhere (e.g. in the help)). Furthermore, it would be nice to define the file base name via an commandline command argument.